apisupport/maven.apisupport/src/org/netbeans/modules/maven/apisupport/MavenNbModuleImpl.java

```diff
@@ -119,13 +119,13 @@ static List<RepositoryInfo> netbeansRepo() {
                 RepositoryPreferences.getInstance().getRepositoryInfoById(MAVEN_CENTRAL));
     }
 
+    /**
+     * Returns the latest known version of the NetBeans maven plugin.
+     * This method will not wait for the index to be downloaded, it will return a default value instead.
+     */
     public static String getLatestNbmPluginVersion() {
         RepositoryQueries.Result<NBVersionInfo> versionsResult = RepositoryQueries.getVersionsResult(GROUPID_APACHE, NBM_PLUGIN, null);
 
-        if (versionsResult.isPartial()) {
-            versionsResult.waitForSkipped();
-        }
```

Comment:
@matthiasblaesing lets see if this works. Its now essentially doing what you suggested - assuming all tests are green.

Reply:
did some more manual testing and everything worked fine. I think we could try to get this in NB 14 (assuming travis is green again, queue is full ATM)

This PR would reduce on-disk storage size by about 800mb for a fresh index and queries appear to have slightly better performance too when tested side by side.